### PR TITLE
Merge into main

### PR DESCRIPTION
**Pull request by micahcarroll**
Key Changes
Added new experiment config files in config/experiment_configs/
Updated root.py and added data_root.py for better path management

Updated environment_queue.py to be trajectory based rather than subenv-init state based so that multiple gpus can generate trajs for the same subenv-init state 

Added OpenAI API support
Implemented openai_backend.py for GPT model integration
Created openai_finetuning.py for model fine-tuning

Updated experiment configurations
Added new YAML files for various experiment setups (EI, KTO, OpenAI)

Updated prep_for_slurm.py to fix fissues

Enhanced logging and visualization
Updated wandb_logging.py for improved experiment tracking
Added utils_pandas.py for DataFrame operations. Should fix average error we were having

Added unit tests
Created test_basics.py for configuration validation

### CODE REVIEW

File: influence_benchmark/backend/hf_backend.py
Comment (micahcarroll) [reply influence_benchmark/backend/hf_backend.py]: Done, I think

File: influence_benchmark/RL/openai_finetuning.py
Comment (micahcarroll) [inline influence_benchmark/RL/openai_finetuning.py:97]: Try/catch

Comment (carolius) [reply influence_benchmark/RL/openai_finetuning.py:97]: should have done this in new PR

**Comment by micahcarroll** [comment]
We should merge this once #105 and #106 have been merged, but let's not tack on anything other than those for now.

**Review by carolius — APPROVED** [review]
Looks good

**Pull request merged**
